### PR TITLE
Add Grid-on-Sawtooth example

### PR DESCRIPTION
**Pull request by dnewh**
This adds a walkthrough for setting up Grid-on-Sawtooth and creating a schema and product.

### CODE REVIEW

**Review by rbuysse — DISMISSED** [review]

File: examples/sawtooth/README.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# Running Hyperledger Grid on Splinter
```
Comment (adeebahmed) [inline examples/sawtooth/README.md:1]: Should this say `Running Hyperledger Grid on Sawtooth`?

Comment (dnewh) [reply examples/sawtooth/README.md]: Yes, updated.

File: examples/sawtooth/README.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
+# Running Hyperledger Grid on Sawtooth
```
Comment (dplumb94) [inline examples/sawtooth/README.md:1]: I think we should remove this doc and then just link to the website. That is what we will be doing with the Splinter example.

**Review by dplumb94 — CHANGES_REQUESTED** [review]

**Review by dplumb94 — CHANGES_REQUESTED** [review]
getting this no response error from heartbeat: 

```
sawtooth-settings-tp         | INFO  | sawtooth_sdk::proces | Message: 42415b52b7444c4dae7c23f0f630290c
sawtooth-settings-tp         | INFO  | sawtooth_sdk::proces | sending PingResponse
sawtooth-sabre-tp            | 2020-11-04 16:59:30,744 INFO  [sawtooth_sdk::processor] Message: e6aff8788bc744e4806bc90b3e7fc1be
sawtooth-sabre-tp            | 2020-11-04 16:59:30,744 INFO  [sawtooth_sdk::processor] sending PingResponse
sawtooth-validator           | [2020-11-04 16:59:40.707 DEBUG    interconnect] No response from 1c0905a7f6b12ed9e4f3abf1faf90a1c211ecd5ca1ddafaae58eb4d880dabec329589abd0bbacae24d6367c61d68dc07bc41a107b3f279cb412b4bc5c2ef750f in 19.966135263442993 seconds - beginning heartbeat pings.
sawtooth-validator           | [2020-11-04 16:59:50.717 DEBUG    interconnect] No response from ec4aea75b9ad7e8b02559fc691f4f7e1442d09aab9f1621e6d6197beb7ed0130fc6258fceeebb425ea59ba15c968c48900c74898790c44b69198a0345d8c8180 in 19.974361181259155 seconds - beginning heartbeat pings.
```
Also got this error submitting the org:
```
root@gridd-alpha:/#  grid organization create \
> 314156 myorg '123 main street' \
> --metadata gs1_company_prefixes=314156
Reqwest Error: https://localhost:8080/batches: error trying to connect: error:1408F10B:SSL routines:ssl3_get_record:wrong version number:../ssl/record/ssl3_record.c:332:
```

File: examples/sawtooth/README.md (hunk 2 above)
Comment (dnewh) [reply examples/sawtooth/README.md:1]: updated

**Review by dplumb94 — APPROVED** [review]

**Review by RyanLassigBanks — APPROVED** [review]

**Pull request merged**